### PR TITLE
new record

### PR DESCRIPTION
**Pull request by thea-m**

### CODE REVIEW

File: new/PRS13031WaldaA.xml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        <revisionDesc>
+            <change who="DR" when="2020-06-08">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ወልደ፡ አቢብ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Walda ʾAbib</persName>
```
Comment (SusanneHummel) [inline new/PRS13031WaldaA.xml:49]: Gibt es nach Walda ein Leerzeichen zuviel?

Comment (MassimoVilla) [reply new/PRS13031WaldaA.xml:49]: it doesn't seem to me

File: new/PRS13031WaldaA.xml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+                </person>
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="PRS13031WaldaA" passive="BDLaethd20"/>
```
Comment (MassimoVilla) [inline new/PRS13031WaldaA.xml:52]: Might it be useful to add a short comment about the relation inside <desc/>? (e.g., His name is written in the supplication formulas etc...)

Comment (thea-m) [reply new/PRS13031WaldaA.xml:52]: @MassimoVilla you are right. I will do it for all the persons attested in this manuscript together later! (Then a floruit can also be added)

**Pull request merged**